### PR TITLE
following of PR 1038

### PR DESCRIPTION
**Pull request by jeromebeguin**

### CODE REVIEW

**Comment by poeschlr** [comment]
Could you provide a description of what you changed?
Links to datasheets, screenshots, a list of what symbols you added, ...

**Comment by jeromebeguin** [comment]
I've resolved the conflicts of the previous PR. Nothing more.

**Comment by poeschlr** [comment]
And you don't even provide a link to the previous pull request?
For everyone else, here the link: #1038

It seems the following symbols are changed (Changed description and keywords. Added a datasheet link. Changed pin positions.)
TPS62130  with aliases TPS62131, TPS62132 and TPS62133

**TPS62130**:
screenshot from 2017-04-10 12-42-16
Old symbol
screenshot from 2017-04-10 12-44-44

It would really help if you could provide this information in future pull requests. The maintainers have a lot of pull requests to look at. (And don't necessary remember which pull request had what problems.) 
If everyone would provide all the information required in the pull request, the merge process would be a lot faster. (And there maybe would not be 100+ open pull requests.)
What information would be nice:
* What symbols have been changed.
* What has changed in the symbols.
* A link to the datasheet that has been used to design the symbols.
* Screenshots of the symbols. (Even better if the pin table is included.)

If you include this information it is also a way to check if nothing went wrong when creating the pull request. Because if there are changes in the files that are not listed in the description, something clearly went wrong.

**Comment by jeromebeguin** [comment]
Ok, I got it. I'll do this way next time. Thanks for the informations.

**Comment by jkriege2** [comment]
What is the intention of these changes? 

Also on the TPS62130  you have SW with types PASSIVE and OUTPUT??? Why not only one type? These pins seem to be internally connected, so they should be all the same (you could even stack all the SW pins I think! ... also PVIN and PGND could be stacked ... just a suggestion though ;-) 

Best,
JAN

**Comment by SchrodingersGat** [comment]
@jkriege2 I believe it is because if you have multiple OUTPUTS that you intend to connect together, they will not comply with ERC...

KiCad *desperately* needs multi-pin connectivity...

**Comment by jkriege2** [comment]
@SchrodingersGat : But then it should be consistent over both symbols (at least ;-)

How should we handly this until new features get available?

Best,
JAN

**Comment by poeschlr** [comment]
I think there is only one symbol added. (with aliases)

The screenshots i added show the old symbol and the new symbol (to highlight what changed.)

**Comment by jeromebeguin** [comment]
@jkriege2 The OUT/PASSIVE/PASSIVE for SW pins in TPS62130 is because these pins should be connected together in the schematics. If more than one is "OUT", It makes an error because two output connected together. It's just a way to avoid the error in EESCHEMA.

I'm sorry I don't understand what you mean by "changed VCC from W to w"
Pin table of LM5175RHF: 
image

File: library/texas.lib [hunk 1]
```diff
@@ -432,7 +432,7 @@ X LDRV2 21 700 -500 200 L 50 50 1 1 O
 X VOSNS 12 700 500 200 L 50 50 1 1 O
 X PGND 22 -200 -1000 200 U 50 50 1 1 W
 X ISNS(-) 13 700 -600 200 L 50 50 1 1 I
-X VCC 23 700 700 200 L 50 50 1 1 W
+X VCC 23 700 700 200 L 50 50 1 1 w
```
Comment (poeschlr) [inline library/texas.lib]: @jeromebeguin 
> I'm sorry I don't understand what you mean by "changed VCC from W to w"

@jkriege2 meant this change.

**Comment by jeromebeguin** [comment]
I have no idea what this is, I'll investigate.

**Comment by evanshultz** [comment]
@jkriege2 @poeschlr 
Is there anything left to do on this symbol? Can it be merged?

**Comment by evanshultz** [comment]
Sorry for the traffic. GH was having issues.

@jkriege2 @poeschlr
Is there anything left to do on this symbol? Can it be merged? It seems like the last comment above was implemented and Travis doesn't find any issues.

**Comment by jkriege2** [comment]
Hi!

sorry for the long wait in review ... I havre a few suggestions:
1. I think this symbol should go to dc-dc.lib
2. one could stack the input7output pins in this symbol to clean it up a bit
3. Finally I would change the pin length to 100mil ... that should be enough to accomodate the numbers and makes the symbol a bit sleeker (IMHO).

Otherwise the symbol(s) look fine.

Best,
JAN

**Comment by evanshultz** [comment]
@jkriege2 Thanks for looking! With the imminent reorganization, is moving this symbol necessary?

I'm assuming this PR is abandoned since the originator closed it. If I don't hear anything, I'll make the updates above and submit a new PR.

**Comment by jkriege2** [comment]
@jeromebeguin Are yu still interested in this PR?

JAN

**Comment by jeromebeguin** [comment]
I'm sorry I don't have the time anymore for following the PR.

**Comment by jkriege2** [comment]
Sorry to hear that ...

So should I close this PR? May we (potentially) take the symbols you posted, modify and merge them ... if we find a volunteer?

best,
JAN

**Comment by jeromebeguin** [comment]
As you wish, the symbol is here and available to whoever want to do something with it. 

Best

Jérôme

**Comment by evanshultz** [comment]
I'll take this one and open a new PR soon.

**Comment by evanshultz** [comment]
@jkriege2 PR submitted at https://github.com/KiCad/kicad-library/pull/1552.

**Pull request closed without merging**